apps/ledger-live-desktop/src/renderer/modals/SignMessage/Body.tsx

```diff
@@ -39,9 +42,19 @@ const steps: Array<St> = [
     },
   },
 ];
+
+type OwnProps = {
+  onClose?: () => void;
+  data: Data;
+};
+type Props = OwnProps;
+
 const Body = ({ onClose, data }: Props) => {
+  const steps = useMemo(() => {
+    return data.message.standard === "EIP712" ? eip712Step : messageSteps;
+  }, [data.message.standard]);
```

Comment:
nit pick: now that you moved the array creation outside the component, IMO we can remove the useMemo as it doesn't provide more value now, not a big deal if you keep it too ofc